example.c

```diff
@@ -0,0 +1,20 @@
+#include "header.h"
+
+int leak_example(int c) {
+  void *p = my_malloc(10);
+  if(c)
+    return -1;   // "p" is leaked
+  /* ... */
+  my_free(p);
+  return 0;
+}
+
+
+int leak_example1(int c) {
```

Comment:
example.c:13: CID 312465 (#1 of 1):
  Type: Resource leak (RESOURCE_LEAK)
  Classification: Unclassified
  Severity: Unspecified
  Action: Undecided
  Owner: Unassigned
  First detected on 2018-01-29.
 example.c:11:
  1. alloc_fn: Storage is returned from allocation function "my_malloc".
 example.c:3:
  1.1. alloc_fn: Storage is returned from allocation function "malloc".
 example.c:3:
  1.2. return_alloc_fn: Directly returning storage allocated by "malloc".
 example.c:11:
  2. var_assign: Assigning: "p" = storage returned from "my_malloc(10)".
 example.c:12:
  3. path: Condition "c", taking true branch.
 example.c:13:
  4. leaked_storage: Variable "p" going out of scope leaks the storage it points to.